sushy/redfish_interop_profiles/ServiceRoot.v1_0_1.json

```diff
@@ -0,0 +1,75 @@
+{
+    "SchemaDefinition": "OpenstackSushyRedfishInteroperabilityProfile.v1_0_0",
+    "ProfileName": "ServiceRootProfile",
+    "ProfileVersion": "1.0.1",
+    "Purpose": "Specifies the OpenStack sushy Redfish client requirements of a Redfish service implementation, typically offered by a baseboard management controller (BMC)",
+    "OwningEntity": "Dell Inc.",
+    "ContactInfo": ["richard.pioso@dell.com", "shrivastava_pranjali@dellteam.com", "digmabar_patil@dellteam.com"],
+    "Protocol": {
+        "MinVersion" : "1.0.0",
```

Comment:
nit: Please remove the space before the colon -- `s/ :/:/`

This comment applies to other locations in the file.